Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -140,9 +140,9 @@ public SeedData Cross(SeedData a, SeedData b)
     private void MutateFloat(ref float val, float min, float max, int bits, int totalbits, float mult)
     {
         // Probability that a bit flip happens for this value.
-        float p = mult * bits / totalbits;
-        p = Math.Clamp(p, 0, 1);
-        if (!Random(p))
+        float probBitflip = mult * bits / totalbits;
```

Comment:
I also think this is fine, but *p* is mathematical convention and this convention has existed long before computers have.